### PR TITLE
feat: implement cors

### PR DESCRIPTION
**Pull request by JakubKotkowiak**

### CODE REVIEW

File: src/functions/get-validation-history.ts [hunk 1]
```diff
@@ -1,68 +1,104 @@
...
-
-  if (params.accountId && params.searchPhrase) {
-    return LambdaResponses.badRequest;
+  let origin = undefined;
+  if (event.headers.origin) {
+    origin = [event.headers.origin];
+  } else {
+    origin = [""];
   }
+  let allowed = "Denied";
```
Comment (DominikMarciniszyn) [inline src/functions/get-validation-history.ts:28]: I prefer to move it to the const variable

File: src/functions/get-lookup-response.ts [hunk 2]
```diff
@@ -5,42 +5,67 @@ import { LambdaResponse } from "../types/response";
...
-  if (!params.orgId) {
-    logger.error("No org id was given!");
-    return LambdaResponses.badRequest;
+  let origin = undefined;
+  if (event.headers.origin) {
+    origin = [event.headers.origin];
+  } else {
+    origin = [""];
   }
+  let allowed = "Denied";
```
Comment (DominikMarciniszyn) [inline src/functions/get-lookup-response.ts:22]: I prefer to move it to the const variable

File: src/functions/manual-approval.ts [hunk 3]
```diff
@@ -14,46 +16,75 @@ export const manualApprove = async (
   context: Context
 ): Promise<LambdaResponse> => {
   const logger = new Logger(context);
+  let origin = undefined;
+  if (event.headers.origin) {
+    origin = [event.headers.origin];
+  } else {
+    origin = [""];
+  }
+  let allowed = "Denied";
```
Comment (DominikMarciniszyn) [inline src/functions/manual-approval.ts:25]: I prefer to move it to the const variable

**Review by DominikMarciniszyn — CHANGES_REQUESTED** [review]

**Review by DominikMarciniszyn — APPROVED** [review]

File: src/functions/get-validation-history.ts (hunk 1 above)
Comment (JakubKotkowiak) [reply src/functions/get-validation-history.ts:30]: It's a let since it's being reassigned later.

**Pull request merged**